internal/manifest/oci_index_test.go

```diff
@@ -50,6 +51,61 @@ func TestOCI1IndexFromManifest(t *testing.T) {
 	testValidManifestWithExtraFieldsIsRejected(t, parser, validManifest, []string{"config", "fsLayers", "history", "layers"})
 }
 
+func TestOCI1EditManifestt(t *testing.T) {
+	validManifest, err := os.ReadFile(filepath.Join("testdata", "ociv1.image.index.json"))
+	require.NoError(t, err)
+	list, err := ListFromBlob(validManifest, GuessMIMEType(validManifest))
+	require.NoError(t, err)
+	editInstances := []ListEdit{}
+	editInstances = append(editInstances, ListEdit{
+		UpdateOldDigest: list.Instances()[0],
+		UpdateDigest:    "sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb",
+		UpdateSize:      32,
+		UpdateMediaType: "something",
+		ListOperation:   ListOpUpdate})
+	err = list.EditInstances(editInstances)
+	require.NoError(t, err)
+	assert.Equal(t, list.Instances()[0], editInstances[0].UpdateDigest, "editInstanceTest")
+	instance, err := list.Instance(list.Instances()[0])
+	require.NoError(t, err)
+	assert.Equal(t, instance.MediaType, "something", "editInstanceTest")
+	assert.Equal(t, instance.Size, int64(32), "editInstanceTest")
+
+	// Verfiy correct zstd sorting
```

Comment:
Please start with a fresh
```go
	list, err = ListFromBlob(validManifest, GuessMIMEType(validManifest))
	require.NoError(t, err)
```

here, so that the data from this operation is unaffected by the previous test.

Reply:
Done.